### PR TITLE
add get daily participant reward transfers

### PR DESCRIPTION
**Pull request by juliangruber**
- [x] Make address a path parameter
- [x] Add tests

### CODE REVIEW

**Comment by juliangruber** [comment]
@bajtos hindsight please 🙏 (need to make progress with Desktop v2)

File: stats/lib/handler.js [hunk 1]
```diff
@@ -92,6 +93,8 @@ const handler = async (req, res, pgPools) => {
     await respond(fetchParticipantChangeRates)
   } else if (req.method === 'GET' && segs[0] === 'participant' && segs[1] && segs[2] === 'scheduled-rewards') {
     await respond(fetchParticipantScheduledRewards, segs[1])
+  } else if (req.method === 'GET' && segs[0] === 'participant' && segs[1] && segs[2] === 'reward-transfers') {
```
Comment (bajtos) [inline stats/lib/handler.js:96]: I think this data is generic (shared by all impact evaluators), it isn't specific to Spark. Therefore, the implementation should go to `stats/lib/platform-routes.js` and `stats/lib/platform-stats-fetcher.js`.

File: stats/lib/stats-fetchers.js [hunk 2]
```diff
@@ -148,6 +148,20 @@ export const fetchParticipantScheduledRewards = async (pgPools, { from, to }, ad
...
+/**
+ * @param {PgPools} pgPools
+ * @param {import('./typings.js').DateRangeFilter} filter
+ * @param {string} address
+ */
+export const fetchParticipantRewardTransfers = async (pgPools, { from, to }, address) => {
+  const { rows } = await pgPools.stats.query(`
+    SELECT day::TEXT, amount
+    FROM daily_reward_transfers
+    WHERE to_address = $1 AND day >= $2 AND day <= $3
```
Comment (bajtos) [inline stats/lib/stats-fetchers.js:160]: Could you please check that we have an SQL index to make this query performant? I think the index should be over `to_address` first and `day` second.

**Review by bajtos — COMMENTED** [review]
The changes look mostly good 👍🏻 

I have two comments to address.

File: stats/lib/handler.js (hunk 1 above)
Comment (juliangruber) [reply stats/lib/handler.js:96]: https://github.com/filecoin-station/spark-stats/pull/178

File: stats/lib/stats-fetchers.js (hunk 2 above)
Comment (juliangruber) [reply stats/lib/stats-fetchers.js:160]: We have these:

```sql
CREATE INDEX daily_reward_transfers_day ON daily_reward_transfers (day);
CREATE INDEX daily_reward_transfers_last_block ON daily_reward_transfers (last_checked_block DESC);
```

Will add

**Pull request merged**